CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ Changelog
 
 See also the [GitHub releases page](https://github.com/FriendsOfSymfony/FOSHttpCache/releases).
 
+2.15.4
+------
+
+* Add flag `fallbackToSmaxage` to `CustomTtlListener` to allow controlling fallback to s-maxage if custom TTL header is not defined on the response.
```

Comment:
```suggestion
2.16.0
------

* Add flag `fallbackToSmaxage` to `CustomTtlListener` to allow controlling fallback to s-maxage if custom TTL header is not defined on the response.
```

Do you want to tag this as a new minor with the new flag? @dbu

Reply:
yes. i am editing the changelog locally and will squash the commits.

Reply:
👍